### PR TITLE
PSTK-1362 - Query Backfill

### PR DESCRIPTION
**Pull request by mfattoru**
Implementation of [query backfill proposal](https://github.com/washingtonpost/ans-schema/pull/231)

### CODE REVIEW

File: src/main/resources/schema/ans/0.10.5/collection.json [hunk 1]
```diff
@@ -173,6 +173,9 @@
 
     "content_aliases": {
       "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.5/traits/trait_content_aliases.json"
+    },
+    "dynamic_items": {
+      "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.5/traits/trait_adynamic_items.json"
```
Comment (meg-delaunay) [inline src/main/resources/schema/ans/0.10.5/collection.json:178]: i think there's a typo in there, `trait_adynamic_items`

Comment (mfattoru) [reply src/main/resources/schema/ans/0.10.5/collection.json]: nice catch. Thanks

**Comment by brendaneamon** [comment]
Please don't merge until a WebSked team member has had a chance to review.

File: src/main/resources/schema/ans/0.10.5/traits/trait_dynamic_items.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "type": {
+      "title": "Type",
+      "description": "Section or Tag for query backfilling",
+      "type": "string",
+      "enum": ["sections", "tags"]
+    },
+    "ids": {
+      "description": "A list of Section or Tag IDs that will be used to Backfill queries.",
+      "type": "array",
+      "maxItems": 100,
```
Comment (brendaneamon) [inline src/main/resources/schema/ans/0.10.5/traits/trait_dynamic_items.json:17]: Assuming my emended comment on the proposal is accurate, this is probably OK: https://github.com/washingtonpost/ans-schema/pull/231#discussion_r472453352

Comment (mfattoru) [reply src/main/resources/schema/ans/0.10.5/traits/trait_dynamic_items.json:17]: yes, that's the right understanding

**Comment by mfattoru** [comment]
can this be merged?

**Comment by brendaneamon** [comment]
I'm good with merging if it looks OK to @meg-delaunay.  I don't think Joseph has any more comments, so I'll remove the request for review from him.

**Comment by meg-delaunay** [comment]
Can we either add these fields to https://github.com/washingtonpost/ans-schema/blob/master/tests/fixtures/schema/0.10.5/collection-fixture-good.json or create a new test fixture as part of this? Just want it documented in there somewhere as an example at the very least.

**Pull request merged**